config/iam-components.yaml

```diff
@@ -0,0 +1,1152 @@
+postgresql:
+  fullnameOverride: postgresql-iam
+  auth:
+    username: keycloak
+    enablePostgresUser: true
+    existingSecret: postgres-secret
+    secretKeys:
+      adminPasswordKey: postgres-root-password
+      userPasswordKey: postgres-password
+  rbac:
+    create: false
+  primary:
+    persistence:
+      enabled: false
+    initdb:
+      scripts:
+        create.sh: |
+          psql postgresql://postgres:${POSTGRES_PASSWORD}@localhost:5432 -c "CREATE DATABASE keycloak_iam;"
+          psql postgresql://postgres:${POSTGRES_PASSWORD}@localhost:5432 -c "ALTER DATABASE keycloak_iam OWNER TO keycloak;"
+
+mysql:
+  nameOverride: mysql-iam
+  fullnameOverride: mysql-iam
+  auth:
+    existingSecret: mysql-secret
+  serviceAccount:
+    create: false
+  primary:
+    persistence:
+      enabled: false
+  secondary:
+    persistence:
+      enabled: false
+  initdbScripts: {}
+
+vcwaltid:
+  ingress:
+    enabled: false
+
+  fullnameOverride: waltid-iam
+  deployment:
+    image:
+      repository: quay.io/fiware/waltid
+      tag: 1.14.1-SNAPSHOT
```

Comment:
This applies also to the other components:
Shouldn't the IAM chart already set the correct image tags and registries to be used? In this case those parameters can be removed.

Reply:
yes, you're right, I'll update it